converter/output/rita/constants/constants.go

```diff
@@ -0,0 +1,16 @@
+package constants
```

Comment:
Since the strobes code is separate from the rita db manager code, and both sets of code needed access to these constants, a new package was created to hold them. This prevents import cycles.